docs/get-started/features.md

```diff
@@ -134,6 +134,17 @@ examples how to set-up and manage the worker configuration.
 To aid in making Custom Features clearer, we define a general and a per rule
 nomenclature, keeping things as consistent as possible.
 
+#### Additional configuration directory
+
+Additionally to the rules defined in the nfd-worker configuration file, the
+Custom feature can read morddde configuration files located in the
```

Comment:
>  morddde
typo? must have missed it last round of review.

Reply:
You didn't miss it, I messed it up somehow during reformat in the latest push. Thx and Fixed.